.travis.yml

```diff
@@ -16,10 +16,13 @@ git:
   depth: 1000000
 
 before_install:
-  # Install the 32-bit cross compiler
-  - wget http://releases.linaro.org/components/toolchain/binaries/5.3-2016.02/arm-linux-gnueabihf/gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf.tar.xz
-  - tar xf gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf.tar.xz
-  - export PATH=${PWD}/gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf/bin:${PATH}
+  # Install the 32-bit cross compiler. We're using an older version here (4.9)
+  # because some Android setups still use this version which can cause build
+  # errors not caught by later GCC versions, see
+  # https://github.com/OP-TEE/optee_os/issues/1998.
+  - wget http://releases.linaro.org/archive/15.02/components/toolchain/binaries/arm-linux-gnueabihf/gcc-linaro-4.9-2015.02-3-x86_64_arm-linux-gnueabihf.tar.xz
```

Comment:
This release is a bit old, any reason why this particular release was selected?

Reply:
No particular reason other than it was the one we used previously
```
$ git log --oneline -n 1 -Sgcc-linaro-4.9
0806ea94 Update travis for gcc5
```
I can pick another one though. Maybe the exact one that's used to build AOSP? @vchong?

Reply:
I understand why we are doing this, but it feels like the wrong way to go here, to downgrade the toolchains for all builds. Wouldn't it be better to upgrade to gcc7.x for the majority of the builds and then keep one sanity build using gcc4.9? Sure, we would need to download two toolchains, but I think that could be worth it.

Reply:
These days this is only a single build. Most of the builds are done in shippable.

Reply:
> Maybe the exact one that's used to build AOSP? 

It's here [1], but please note that the actual gcc binary is `real-arm-linux-androideabi-gcc`, not `arm-linux-androideabi-gcc`.

[1] https://android.googlesource.com/platform/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/+archive/master.tar.gz